x-pack/plugins/ingest_manager/common/types/models/epm.ts

```diff
@@ -29,6 +29,7 @@ export enum KibanaAssetType {
   search = 'search',
   indexPattern = 'index-pattern',
   map = 'map',
+  'canvas-workpad-template' = 'canvas-workpad-template',
```

Comment:
Can we have this camel-cased here, so `canvasWorkpadTemplate`, following the `indexPattern` two lines above?

Reply:
@skh I _think_ there might be a bug here that prevents this. (But I might be misunderstanding how a package should be structured).  

If the file (in this case) is `package-version/kibana/canvas-workpad-template/whatever.json` then the type will be `canvas-workpad-template` and the `in KibanaAssetType` [check here](https://github.com/elastic/kibana/blob/b435d4608036c89439d1cfec6349ac7bc22adbca/x-pack/plugins/ingest_manager/server/services/epm/registry/index.ts#L192) would fail if KibanaAssetType had the camel cased key.  

So I guess the question is should the types in the package match what's defined in this KibanaAssetType or match the actual Kibana Saved Object Type?

I could not find any example packages that actually had an "index-pattern" or "indexPattern" folder, but the examples in the package-registry repo uses "index-pattern" for the folder name, which would fail to have those assets show up.

Reply:
I'm afraid you're right. Thanks for spotting this, I need to look into it.